VerifiableCredential/index.mjs

```diff
@@ -1,42 +1,9 @@
 import AJV from "ajv";
 import JSONPath from "jsonpath";
-import { LitElement, css, html, nothing } from "lit";
-import { ifDefined } from 'lit/directives/if-defined.js';
-import { map } from 'lit/directives/map.js';
-import { when } from "lit/directives/when.js";
 
 const VERSION = "https://identity.foundation/credential-manifest/spec/v1.0.0/";
```

Comment:
unrelated but is this URL supposed to work? because it doesn't currently

Reply:
according to <[https://identity.foundation/credential-manifest/#versioning]()> it needs to be something along these lines, but it probably should be `v0.0.1` instead 😅